wic/src/main/java/edu/asu/diging/wic/web/HomeController.java

```diff
@@ -118,4 +120,12 @@ private GraphElement createElement(Node node, IConcept concept) {
         }
         return element;
     }
+    
+    @RequestMapping(value="/getAllConceptsTypes", method=RequestMethod.GET)
+    public ResponseEntity<Collection<ConceptType>> getAllConceptsTypes(Model model) {
```

Comment:
since this returns json, there is no need for a model (remove parameter and line 128).